cmake/defaults/msvcdefaults.cmake

```diff
@@ -78,6 +78,16 @@ if (NOT Boost_USE_STATIC_LIBS)
     _add_define("BOOST_ALL_DYN_LINK")
 endif()
 
+if(${MAYAUSD_DEFINE_BOOST_DEBUG_PYTHON_FLAG})
+    # We should not have to add this define manually as it is added by the
+    # compiler when you specify the /MTd or /MDd option. But it is not - needs
+    # to be investigated.
+    _add_define(_DEBUG)
+
+    _add_define(BOOST_DEBUG_PYTHON)
+    _add_define(BOOST_LINKING_PYTHON)
+endif()
+
```

Comment:
We need all these extra defines with building with a debug python. I still need to figure out why _DEBUG is not being added automatically as it should be. It is needed when including the python headers (on Windows) so that the proper #import statement is used.

Reply:
I get that _DEBUG is a msvc thing... but are BOOST_DEBUG_PYTHON and BOOST_LINKING_PYTHON as well?

Reply:
Ah you are right, they are not. I'll move those two into the CXXDefaults.cmake file.